### PR TITLE
DB Seeding

### PR DESCRIPTION
**Pull request by beazley2**
DB seeding added for easier testing.

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -4,21 +4,34 @@
...
+#  database: db/development.sqlite3
 
 # Warning: The database defined as "test" will be erased and
 # re-generated from your development database when you run "rake".
 # Do not set this db to the same as development or production.
+#test:
+#  <<: *default
+#  database: db/test.sqlite3
+
+development:
```
Comment (ardeshirj) [inline config/database.yml]: I think it's nice to use sqlite for the development and test database, so we won't mess around with production (aws) database during the development. We can talk more about this in our next meeting. If we ended up keeping the sqlite we also need to add sqlite gem back to Gemfile

Comment (beazley2) [inline config/database.yml]: The way I have it, it sets up two additional local postgres DBs for testing and development accessible via PGAdmin, so it doesn't touch the AWS one.

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -20,7 +20,7 @@ def create
   private
 
     def user_params
-      params.require(:user).permit(:name, :email, :password,
+      params.require(:user).permit(:name, :email, :location, :password,
```
Comment (sector46) [inline app/controllers/users_controller.rb]: Do we want to save the user's name as separate entities (first, last) or just as one concatenated field?

File: app/views/users/new.html.erb [hunk 3]
```diff
@@ -15,6 +15,11 @@
     </div>
 
     <div class='form-group'>
+      <%= f.label :location, class: 'col-sm-2 control-label' %>
```
Comment (sector46) [inline app/views/users/new.html.erb]: 'f' is left ambiguous in this version. Do we want to use 'formElement' as Ardeshir's latest pull request suggests?

File: app/controllers/users_controller.rb (hunk 2 above)
Comment (beazley2) [inline app/controllers/users_controller.rb]: I think Ankur has them split up on the web page so probably that. It's on the to-do list.

File: app/views/users/new.html.erb (hunk 3 above)
Comment (beazley2) [inline app/views/users/new.html.erb]: Yeah, I fixed this in a later commit. (Currently about 6 commits ahead locally)

**Pull request merged**